tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -1055,6 +1054,15 @@ def is_element_displayed(self, by, locator):
     def tearDown(self):
         if self.device.is_desktop_b2g and self.device.storage_path:
             shutil.rmtree(self.device.storage_path, ignore_errors=True)
+        if self.device.is_android_build:
+            file_items = self.device.file_manager.list_items('/data/local/webapps/')
+            dirs_to_keep = ['{14750155-5f3d-41ab-a527-f16adf4c3293}', 'marketplace-dev.allizom.org', 'marketplace.allizom.org', 'marketplace.firefox.com']
```

Comment:
I just noticed that `"removable": true` also applies to the Mochitest and Marketplace apps, so I don't think we can use that. Perhaps there's something else?